core/primitives/src/contract.rs

```diff
@@ -1,21 +1 @@
-use crate::hash::{hash as sha256, CryptoHash};
-
-pub struct ContractCode {
-    pub code: Vec<u8>,
-    pub hash: CryptoHash,
-}
-
-impl ContractCode {
-    pub fn new(code: Vec<u8>, hash: Option<CryptoHash>) -> ContractCode {
-        let hash = hash.unwrap_or_else(|| sha256(&code));
-        ContractCode { code, hash }
-    }
-
-    pub fn get_hash(&self) -> CryptoHash {
-        self.hash
-    }
-
-    pub fn get_code(&self) -> &Vec<u8> {
-        &self.code
-    }
-}
+pub use near_primitives_core::contract::*;
```

Comment:
why do we do this? Is there any point in keeping this file?

Reply:
This was so that other crates that depend on `near-primitives` don't have to depend on `near-primitives-core`.

Reply:
Okay, so why do we have near-primitives-core in the first place

Reply:
To remove duplicated code.  This way the VM crates can depend on this crate and so can `near-primitives`.